examples/component/src/Main.purs

```diff
@@ -19,5 +18,5 @@ main = do
   case container of
     Nothing -> throw "Container element not found."
     Just c  ->
-      let app = element Container.component {}
+      let app = Container.render
```

Comment:
I added `toReactComponent`. It ends up just being an unsafe coerce, but it’s still an important distinction because ReactComponent reference equality matters while `props -> JSX` reference equality does not (meaning no type class wrappers)